api/src/main/java/com/epam/pipeline/controller/user/UserController.java

```diff
@@ -404,4 +405,26 @@ public void exportUsers(@RequestBody PipelineUserExportVO attr, HttpServletRespo
         final MultipartFile file = consumeMultipartFile(request);
         return Result.success(userApiService.importUsersFromCsv(createUser, createGroup, createMetadata, file));
     }
+
+    @PostMapping("/users/{id}/runners")
+    @ResponseBody
+    @ApiOperation(
+            value = "Adds runners to user",
```

Comment:
I'd suggest to call this methods and underlying method in `apiService` - `updateRunners`, as it allows not only add but also remove entities.